common/modules/olm/manifests/1.5.1/amq-streams.v1.5.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1239 @@
+
+    * **Includes Kafka Connect** - Allows for configuration of common data sources and sinks to move data into and out of the Kafka cluster.
+
+    * **Topic Management** - Creates and manages Kafka Topics within the cluster.
+
+    * **User Management** - Creates and manages Kafka Users within the cluster.
+
+    * **Includes Kafka Mirror Maker 1 and 2** - Allows for morroring data between different Apache Kafka® clusters.
+
+    * **Includes HTTP Kafka Bridge** - Allows clients to send and receive messages through an Apache Kafka® cluster via HTTP protocol.
+
+    * **Cluster Rebalancing** - Uses built-in Cruise Control for redistributes partition replicas according to specified goals in order to achieve the best cluster performance.
+
+    * **Monitoring** - Built-in support for monitoring using Prometheus and provided Grafana dashabords.
+
+    ### Upgrading your Clusters
+    
+    The AMQ Streams operator understands how to run and upgrade between a set of Kafka versions.
+    When specifying a new version in your config, check to make sure you are not using any features that may have been removed.
+    See [the upgrade guide](https://access.redhat.com/documentation/en-us/red_hat_amq/7.7/html-single/using_amq_streams_on_openshift/index) for more information.
+
+    ### Security
+
+    AMQ Streams supports TLS encryption, authentication, and authorization out of the box.
+    Supported authentication methods are TLS client certificates, SCRAM-SHA mechanism based on username and password, and OAuth authentication.
+
+    ### Monitoring
+
+    AMQ Streams has support for Prometheus metrics.
+    Enable metrics in the custom resources and configure your Prometheus server to scrape the metrics from all Kafka and Zookeeper pods.
+    With the integrated Kafka Exporter you have now also advanced metrics about topics, consumer groups and consumer group lag.
+    
+    ### Storage
+
+    An efficient data storage infrastructure is essential to the optimal performance of AMQ Streams.
+    AMQ Streams requires block storage.
+    The use of file storage (for example, NFS) is not recommended.
+
+    AMQ Streams supports three types of data storage:
+
+    * Ephemeral (Recommended for development only)
+
+    * Persistent
+
+    * JBOD (Just a Bunch of Disks, suitable for Kafka only. Not supported in Zookeeper.)
+
+    AMQ Streams also supports advanced operations such as adding or removing disks in Kafka brokers or resizing the persistent volumes (where supported by the infrastructure).
+
+    ### Documentation
+
+    Documentation for the current release can be found on Red Hat [website](https://access.redhat.com/documentation/en-us/red_hat_amq/7.7/html-single/using_amq_streams_on_openshift/index).
+    
+  keywords: ['amq', 'amq-streams', 'kafka', 'messaging', 'kafka-connect', 'kafka-streams', 'data-streaming', 'data-streams', 'streaming', 'streams']
+  version: 1.5.1
+  replaces: amqstreams.v1.5.0
```

Comment:
Talking with the OLM people, we can have:

```
    replaces:  amqstreams.v1.4.1
    skips:
    - amqstreams.v1.5.0
```
So that the install doesn't need to stop at version `1.5.0` if starting at version `1.4.1`.  This will still function properly without this change, it just will be slower from what I understand

[1] https://github.com/operator-framework/operator-lifecycle-manager/blob/0943c4fa7384678cbf2a4a171be476aa5d1327f5/doc/design/how-to-update-operators.md#skipping-updates